tests/integration/tier0/backup-release/test_backup_release.py

```diff
@@ -1,24 +1,55 @@
 import os
-import platform
+
+import pytest
 
 from conftest import SATELLITE_PKG_DST, SATELLITE_PKG_URL
 from envparse import env
 
 
-system = platform.platform()
+def assign_repository_variable(get_system_release, repository=None):
+    """
+    Helper function.
+    Assign correct repository setup to install subscription manager from to their respective distribution.
+    """
+
+    epel7_repository = "ubi"
+    epel8_repository = "baseos"
```

Comment:
If the change that I proposed regarding the name is accepted, then this line would need a change too.